### ISSUE TITLE
System.IO.FileNotFoundException: Could not find file '/config/logs/2020-08-07'.

### ISSUE THREAD

**Issue by mregni** [mregni/EmbyStat#1300]
View details in Rollbar: [https://rollbar.com/UPing/EmbyStat/items/760/](https://rollbar.com/UPing/EmbyStat/items/760/)

```
Traceback (most recent call last):
  File "Interop", line 12, in Interop.ThrowExceptionForIoErrno(Interop+ErrorInfo errorInfo, System.String path, System.Boolean isDirectory, System.Func`2[Interop+ErrorInfo,Interop+ErrorInfo] errorRewriter)
  File "Microsoft.Win32.SafeHandles.SafeFileHandle", line 174, in Microsoft.Win32.SafeHandles.SafeFileHandle.Open(System.String path, Interop+Sys+OpenFlags flags, System.Int32 mode)
  File "System.IO.FileStream", line 371, in System.IO.FileStream..ctor(System.String path, System.IO.FileMode mode, System.IO.FileAccess access, System.IO.FileShare share, System.Int32 bufferSize, System.IO.FileOptions options)
  File "EmbyStat.Services.LogService", line 49, in EmbyStat.Services.LogService.GetLogStream(System.String fileName, System.Boolean anonymous)
  File "EmbyStat.Controllers.Log.LogController", line unknown, in EmbyStat.Controllers.Log.LogController.GetZipFile(System.String fileName, System.Boolean anonymous)
  File "(unknown)", line 153, in lambda_method(System.Runtime.CompilerServices.Closure , System.Object , System.Object[] )
  File "Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor+SyncActionResultExecutor", line unknown, in Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor+SyncActionResultExecutor.Execute(Microsoft.AspNetCore.Mvc.Infrastructure.IActionResultTypeMapper mapper, Microsoft.Extensions.Internal.ObjectMethodExecutor executor, System.Object controller, System.Object[] arguments)
  File "Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker", line 99, in Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
  File "Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker", line 961, in Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker+State& next, Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker+Scope& scope, System.Object& state, System.Boolean& isCompleted)
  File "Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker", line 108, in Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
  File "System.Runtime.ExceptionServices.ExceptionDispatchInfo", line 17, in System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
  File "Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker", line 47, in Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker+ActionExecutedContextSealed context)
  File "Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker", line 961, in Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker+State& next, Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker+Scope& scope, System.Object& state, System.Boolean& isCompleted)
  File "Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker", line 72, in Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
  File "System.Runtime.ExceptionServices.ExceptionDispatchInfo", line 17, in System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
  File "System.Runtime.CompilerServices.TaskAwaiter", line 39, in System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(System.Threading.Tasks.Task task)
  File "System.Runtime.CompilerServices.TaskAwaiter", line 46, in System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(System.Threading.Tasks.Task task)
  File "Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker+<<InvokeNextExceptionFilterAsync>g__Awaited|25_0>d", line 235, in Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker+<<InvokeNextExceptionFilterAsync>g__Awaited|25_0>d.MoveNext()
System.IO.FileNotFoundException: Could not find file '/app/Logs/2020-08-08'.
```

**Comment by mregni** [comment]
Same issue as #1294

Looks like unix systems can't find the logs

**Comment by mregni** [comment]
Should be fixed with #1294
